src/vs/platform/notification/common/notification.ts

```diff
@@ -388,6 +398,16 @@ export interface INotificationService {
 	 * @param filter the filter to use
 	 */
 	setFilter(filter: NotificationsFilter): void;
+
+	/**
+	 * Gets current do not disturb mode.
+	 */
+	getDoNotDisturbMode(): boolean;
```

Comment:
I suggest to change this to a property instead of 2 methods:

```ts
doNotDisturbMode: boolean;
```

With `get`/`set` property accessors, we can write actual code to handle changes to the mode.

And if someone wants to read or change the value:

```ts
notificationService.doNotDisturbMode = true;
if (notificationService.doNotDisturbMode) { ... }
```

Alternatively we could also do this (as you like):

```
readonly isDoNotDisturbMode: boolean;

setDoNotDisturbMode(enabled: boolean): void;
```